Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/DeleteCaseTask.java

```diff
@@ -0,0 +1,850 @@
+            releaseManifestFileLocks();
+            throw ex;
+        }
+        return allLocksAcquired;
+    }
+
+    /**
+     * Deletes the auto ingest job input manifests for the case along with the
+     * corresponding data sources.
+     *
+     * @throws InterruptedException If the thread in which this task is running
+     *                              is interrupted while blocked waiting for a
+     *                              coordination service operation to complete.
+     */
+    @NbBundle.Messages({
+        "DeleteCaseTask.progress.openingCaseDatabase=Opening the case database...",
+        "# {0} - manifest file path", "DeleteCaseTask.progress.parsingManifest=Parsing manifest file {0}...",
+        "# {0} - manifest file path", "DeleteCaseTask.progress.deletingManifest=Deleting manifest file {0}..."
+    })
+    private void deleteAutoIngestInput() throws InterruptedException {
+        SleuthkitCase caseDb = null;
+        try {
+            progress.progress(Bundle.DeleteCaseTask_progress_openingCaseDatabase());
+            logger.log(Level.INFO, String.format("Opening the case database for %s", caseNodeData.getDisplayName()));
+            caseDb = SleuthkitCase.openCase(caseMetadata.getCaseDatabaseName(), UserPreferences.getDatabaseConnectionInfo(), caseMetadata.getCaseDirectory());
+            List<DataSource> dataSources = caseDb.getDataSources();
+
+            if (Thread.currentThread().isInterrupted()) {
+                throw new InterruptedException();
+            }
+
+            boolean allInputDeleted = true;
+            for (Path manifestFilePath : manifestFilePaths) {
+                if (Thread.currentThread().isInterrupted()) {
+                    throw new InterruptedException();
+                }
+
+                final File manifestFile = manifestFilePath.toFile();
+                if (manifestFile.exists()) {
+                    progress.progress(Bundle.DeleteCaseTask_progress_parsingManifest(manifestFilePath));
+                    logger.log(Level.INFO, String.format("Parsing manifest file %s for %s", manifestFilePath, caseNodeData.getDisplayName()));
+                    Manifest manifest = null;
+                    for (ManifestFileParser parser : Lookup.getDefault().lookupAll(ManifestFileParser.class)) {
+                        if (parser.fileIsManifest(manifestFilePath)) {
+                            try {
+                                manifest = parser.parse(manifestFilePath);
+                                break;
+                            } catch (ManifestFileParser.ManifestFileParserException ex) {
+                                logger.log(Level.WARNING, String.format("Error parsing manifest file %s for %s", manifestFilePath, caseNodeData.getDisplayName()), ex);
+                            }
+                        }
+                    }
+                    if (manifest != null) {
+                        if (deleteDataSources(manifest, dataSources)) {
```

Comment:
I believe this should have an else that sets allInputDeleted to false

Reply:
Thanks, corrected flag setting and removal of directory input deletion are in the next commit.